tests/carrier_account_test.go

```diff
@@ -15,6 +15,18 @@ func (c *ClientTests) GenerateCarrierAccount() (*easypost.CarrierAccount, error)
 	return carrierAccount, err
 }
 
+func (c *ClientTests) GenerateCarrierAccountWithCustomWorkflow() (*easypost.CarrierAccount, error) {
```

Comment:
Having these test helper functions aren't actually very helpful (especially for one-offs like this). By moving the test logic outside the test itself, it adds more indirection. Can we keep all test logic inside each test so that it's contained?

Now, if this same logic was used in a half dozen tests, it would make sense, but since that's not the case it really only adds more places to look to learn about a test than it provides value. This is true for your other recent PRs that touch tests.